cfme/common/vm.py

```diff
@@ -495,15 +495,15 @@ def set_ownership(self, user=None, group=None, click_cancel=False, click_reset=F
         cfg_btn('Set Ownership')
         if click_reset:
             action = form_buttons.reset
-            msg_assert = lambda: flash.assert_message_match(
```

Comment:
the switch to partial needs a realignment of the next line to fit the requirements